lib/common/util.js

```diff
@@ -209,6 +209,52 @@ function getType(value) {
 }
 
 /**
+ * Used in an Array iterator usually, this will return the value of a property
+ * in an object by its name.
+ *
+ * @param {string} name - Name of the property to return.
+ * @return {function}
+ *
+ * @example
+ * var people = [
+ *   {
+ *     name: 'Stephen',
+ *     origin: 'USA',
+ *     beenToNYC: false
+ *   },
+ *   {
+ *     name: 'Ryan',
+ *     origin: 'Canada',
+ *     beenToNYC: true
+ *   }
+ * ];
+ *
+ * var names = people.map(prop('name'));
+ * // [
+ * //   'Stephen',
+ * //   'Ryan'
+ * // ]
+ *
+ * var peopleInUSA = people.filter(prop('beenToNYC'));
+ * // [
+ * //   {
+ * //     name: 'Ryan',
+ * //     origin: 'Canada',
+ * //     beenToNYC: true
+ * //   }
+ * // ]
+ */
+function prop(name) {
+  return function(item) {
+    if (name in item) {
+      return item[name];
+    }
+  };
+}
+
+module.exports.prop = prop;
```

Comment:
I've wanted this in our library many times in the past. I figured I'd add it in sooner than later.